src/components/molecules/user-menu/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from "react"
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu"
+import Avatar from "../../atoms/avatar"
+import Button from "../../fundamentals/button"
+import GearIcon from "../../fundamentals/icons/gear-icon"
+import SignOutIcon from "../../fundamentals/icons/log-out-icon"
+import { useNavigate } from "react-router-dom"
+import { AccountContext } from "../../../context/account"
+
+const UserMenu: React.FC = () => {
+  const navigate = useNavigate()
+  const { first_name, last_name, email, handleLogout } =
+    useContext(AccountContext)
+
+  const logOut = () => {
+    handleLogout()
+    navigate("/login")
+  }
+  return (
+    <div className="w-large h-large">
+      <DropdownMenu.Root>
+        <DropdownMenu.Trigger asChild>
+          <div className="w-full h-full cursor-pointer">
+            <Avatar
+              user={{ first_name, last_name, email }}
+              color="bg-fuschia-40"
+            />
+          </div>
+        </DropdownMenu.Trigger>
+        <DropdownMenu.Content
+          sideOffset={5}
+          className="border bg-grey-0 border-grey-20 rounded-rounded shadow-dropdown p-xsmall min-w-[200px] z-30"
+        >
+          <DropdownMenu.Item className="mb-1 last:mb-0">
+            <Button
```

Comment:
question: would it make sense to build something similar to `SidebarMenuItem` here and reuse in other dropdown menus?

Reply:
Good shout, but I'd rather check with Kasper when he's back about philosophy around these things. Have a few similar questions, just don't wanna go around refactoring things opposite to what the desired pattern might be.